ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.15-nightly-x86.yaml

```diff
@@ -166,8 +166,8 @@ tests:
     - chain: openshift-qe-node-density-cni
     workflow: openshift-qe-installer-aws
   timeout: 2h0m0s
-- always_run: false
-  as: data-path-9nodes
+- as: data-path-9nodes
+  cron: 0 22 8,22 * *
```

Comment:
@afcollins QE updated our profiles. This aws-ovn-fips is not needed anymore. The FIPs will be added to GCP. Let's move this to always_run: false.

Reply:
I don't see ovn-fips on the line you commented on , but I believe I moved all of them to always_run: false. Please check